packages/peregrine/lib/talons/CheckoutPage/CheckoutError.js

```diff
@@ -1,11 +1,20 @@
+/**
+ * Other checkout place order mutation errors can include
+ * the following list:
+ *
+ * https://devdocs.magento.com/guides/v2.3/graphql/mutations/place-order.html#errors
+ */
+
 const paymentErrorMessage =
     'Unable to place order: Transaction has been declined. Please try again later.';
 
+const removeGQLTag = rawMessage => rawMessage.replace(/GraphQL error:/, '');
+
 class CheckoutError extends Error {
     constructor(gqlError, ...params) {
         super(params);
         this.name = 'CheckoutError';
-        this.message = gqlError.message;
+        this.message = removeGQLTag(gqlError.message);
```

Comment:
I don't think we have to explicitly strip out the gql tag. There is a property on the graphql errors that is just the message after the colon. Probably something like `graphqlErrors[0].message`. I'd have to debug it to see the specific props but I know it's there.

Reply:
`gqlError.message` is the generic error from the server. Since `graphqlErrors` is an array and each one of them has a `meesage` field, how would we know which one to display? Either we can display all the `graphqlErrors.message`'s  or just show the generic message for the whole error object which is `gqlError.message`.

Reply:
I wonder if `error.message` always equivalent to the message from the first element in `graphQLErrors` with `GraphQL Error`.

[![Image from Gyazo](https://i.gyazo.com/9f210ef668d7c6615e652460283e011f.png)](https://gyazo.com/9f210ef668d7c6615e652460283e011f)

In all other cases that we reference `graphqlErrors` in this codebase we either iterate over each element in the array and handle it separately or we treat the first element in the array as the one we will "handle" either by displaying a relevant message or logging it.